starknet_py/tests/e2e/fixtures/constants.py

```diff
@@ -22,6 +22,8 @@
 
 INTEGRATION_GATEWAY_URL = "https://external.integration.starknet.io"
 
+TESTNET_NODE_URL = "http://188.34.188.184:6060"
+
```

Comment:
Is this URL of our node? 👀

Reply:
🤭

Reply:
Would it be hard to retrieve those from env like we do in [jvm sdk](https://github.com/software-mansion/starknet-jvm/pull/281)?

I don't really think we should be keeping secrets out in the open.